README.md

```diff
@@ -43,3 +59,20 @@ Follow the [./PREREQUISITES.md](./PREREQUISITES.md) instructions to configure a
     * http://localhost:8080/info
 * You can access the MongoDB Admin UI at:
   * http://localhost:8090 
+
+---
+
+## Code Conventions
+
+### Solving tickets
+
```

Comment:
Worth mentioning that the ticket should be converted to an issue beforehand, or is it obvious?

Reply:
I'd say one can't create a branch without converting to issue first, since the ticket number is part of the branch name. So, I'd say we can omit that.